build-tools/packages/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -27,6 +28,23 @@ or logos must follow Microsoft's [Trademark & Brand Guidelines](https://www.micr
 Use of Microsoft trademarks or logos in modified versions of this project must not cause confusion or imply Microsoft sponsorship.
 `;
 
+/**
+ * An array of dependencies that should always use tilde dependency ranges instead of caret.
+ */
+const tildeDependencies = [
+    "@typescript-eslint/eslint-plugin",
```

Comment:
Should this list be in some config files for easy updating?

Reply:
Hmm, yeah, you're right. I'll refactor this.

Reply:
Refactored these config settings into the fluidBuild node of package.json.